media-sound/picard/picard-1.4.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+DISTUTILS_SINGLE_IMPL=1
+DISABLE_AUTOFORMATTING=true
+inherit eutils distutils-r1 readme.gentoo-r1
+
+DESCRIPTION="A cross-platform music tagger"
+HOMEPAGE="https://picard.musicbrainz.org"
+SRC_URI="https://github.com/metabrainz/${PN}/archive/release-${PV}.tar.gz"
```

Comment:
Keep `SRC_URI="http://ftp.musicbrainz.org/pub/musicbrainz/picard/${P}.tar.gz"`. To use GitHub release files, you would need to add ` -> {P}.tar.gz` at the end.